src/common/components/vanilla-modules/forms/form-group-select.js

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react';
+
+import styles from '../../../style/vanilla/css/forms.css';
+
+export default function FormGroupSelect(props) {
+  const { disabled, name, label, placeholder } = props;
+  const id = `ID_${name}`;
+  const status = props.touched ? ( props.valid ? 'p-form-validation is-success' : 'p-form-validation is-error') : null;
+
+  return (
+    <div className={ `${styles['p-form__group']} ${styles[status]}` }>
+      <label htmlFor={ id } className={ styles['p-form__label'] }>
+        { label }
+      </label>
+      <div className={ styles['p-form__control'] }>
+        <select
+          id={ id }
+          name={ props.sensitive ? null : name }
+          data-name={ name }
+          required={ props.required }
+          disabled={ disabled }
+          onChange={ props.onChange }
+          value={ props.selectedOption }
+        >
+          <option value="">{ placeholder }</option>
+          { props.options.map(opt => {
+            return (
+              <option
+                key={ opt }
+                value={ opt }
```

Comment:
Making `value` and option text being the same may not be good in some cases.

For example it's quite easy to imagine a select for a list of countries where we'd like to have options values be iso codes (PL, US, UK) while text being a name of the country.

Putting any arbitrary structure to this data also doesn't seem very flexible.

Maybe the easiest way would be to allow devs to use `<option>` as a child of `<FormGroupSelect>`.

So it would end up as:

```jsx
<FormGroupSelect name='country'>
   <option>-- Optional placeholder --</option>
   <option value="PL">Poland</option>
   <option value="UK">United Kingdom</option>
</FormGroupSelect>
```